### ISSUE TITLE
runs.compare().time as timedelta instead of string?

### ISSUE THREAD

**Issue by mherlich** [guildai/guildai#243]
`runs.compare()` returns a data frame in which column `time` is of type `object` (string). Returning a `timedelta` type would improve usability for the column (e.g. calculating averages). However, I am not sure if there are reasons for keeping it as it is now.

As a workaround I use
```
comp = runs.compare()
comp["time"] = pd.to_timedelta(comp["time"])
```
to convert the column.

**Comment by gar1t** [comment]
Yeah, string is plain wrong here. It should be either an int (seconds) or timedelta, timedelta being better IMO.

I would typically be concerned about an API breaking change here but this is so egregious that I'm tempted to mark it as a bug and just make the change. Thankfully `pd.to_timedelta(x)` works when x is a timedelta.

**Comment by gar1t** [comment]
This fix is available in the current pre-release - 0.7.1.dev6. If you want to confirm this, install Guild using `pip install --pre --upgrade guildai`. Your work-around code above should still work.
